cmd/compute-domain-kubelet-plugin/device_state.go

```diff
@@ -371,7 +371,7 @@ func (s *DeviceState) applyComputeDomainChannelConfig(ctx context.Context, confi
 
 	// Create any necessary ComputeDomain channels and gather their CDI container edits.
 	for _, r := range results {
-		channel := s.allocatable[r.Device].Channel
+		_ = s.allocatable[r.Device].Channel
```

Comment:
We talked about reintroducing this `channel`variable and then folding some of the logic below into a function that takes this as a parameter.